tests/config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Since there are some configurations in tests that may change as per the requirement of the organization using it, I created a json file that is parsed in the testcase class.
But it feels like a overkill now, what do you think about it? 
Is this a good way to go about it or can you advice on another better way?

Reply:
I think is fine!